core/templates/services/questions-list.service.ts

```diff
@@ -34,7 +34,8 @@ import { TruncatePipe } from 'filters/string-utility-filters/truncate.pipe';
 })
 export class QuestionsListService {
   private _questionSummariesForOneSkill: QuestionSummaryForOneSkill[] = [];
-  private _nextCursorForQuestions: string = '';
+  private _nextOffsetForQuestions: number = 0;
+  private _more: boolean = true;
```

Comment:
Instead of having this, you could also have _nextOffsetForQuestions be null if there are no more questions to fetch, and an int with the next offset otherwise. If you opt for that, make sure to add code comments to explain what is going on.

If you keep the boolean, please give it a clearer name that describes exactly what it is representing.

Reply:
Kept it to boolean. Changed the name to `_moreQuestions`